docs/rules/no-extra-parens.md

```diff
@@ -23,6 +23,7 @@ This rule has an object option for exceptions to the `"all"` option:
 * `"conditionalAssign": false` allows extra parentheses around assignments in conditional test expressions
 * `"returnAssign": false` allows extra parentheses around assignments in `return` statements
 * `"nestedBinaryExpressions": false` allows extra parentheses in nested binary expressions
+* `"ignoreJSX": "all|multiline|single-line"` allows extra parentheses around all/multiline/single-line JSX components
```

Comment:
Ah, I actually had a `false` options for this field before, but removed it as it seemed the preferred way of disabling the option was to omit the key.

`"none"` sounds like a good default. I will add it.